src/scss/includes/menu.scss

```diff
@@ -270,7 +276,11 @@ $productDrawerWidth: 744px;
   }
 
   .menu-top {
-    height: 88px;
+    height: auto;
+    padding: $spacing-l;
```

Comment:
Do not hesitate to use `var(--spacing-l)` instead of SCSS vars to use CSS Vars from `qwant-ponents`. 😉